packages/@aws-cdk/aws-cloud9-alpha/test/cloud9.environment.test.ts

```diff
@@ -124,14 +125,43 @@ test('environment owner can be an IAM user', () => {
     imageId: cloud9.ImageId.AMAZON_LINUX_2,
     owner: Owner.user(user),
   });
+
+  const userLogicalId = stack.getLogicalId(user.node.defaultChild as CfnUser);
   // THEN
```

Comment:
```suggestion
  // THEN
  const userLogicalId = stack.getLogicalId(user.node.defaultChild as iam.CfnUser);
```
After removing the redundant `import` statement.